### ISSUE TITLE
Allow the customer to decide if to save the payment source

### ISSUE THREAD

**Issue by elia** [solidusio/solidus_starter_frontend#287]
- [ ] Verify if/how this is compliant with regulations (e.g. GDPR)
- [ ] Understand which should be the default (use a configuration?)
- [ ] Allow deleting saved cards

**Comment by fthobe** [comment]
@elia is this still an issue?

**Comment by fthobe** [comment]
Yes it is. @jarednorman I assume the credit card controller needs a strong touch to do that. It's something we need for obvious reasons here in Europe and I assume the problem here is the difference in abstraction between Braintree and stripe. If you want to give some guidance we can do that next week.

**Comment by fthobe** [comment]
Related to #288
